connector/src/main/java/org/jboss/as/connector/subsystems/jca/JcaExtension.java

```diff
@@ -630,9 +644,19 @@ private void parseDistributedWorkManager(final XMLExtendedStreamReader reader, f
                     }
                     case ELYTRON_ENABLED: {
                         switch (readerNS) {
-                            case JCA_5_0: {
+                            case JCA_6_0:
+                            {
+                                String value = rawElementText(reader);
+                                ((SimpleAttributeDefinition) JcaDistributedWorkManagerDefinition.DWmParameters.ELYTRON_ENABLED.getAttribute()).parseAndSetParameter(value, distributedWorkManagerOperation, reader);
+                                break;
+                            }
+                            case JCA_5_0:
+                            {
                                 String value = rawElementText(reader);
                                 ((SimpleAttributeDefinition) JcaDistributedWorkManagerDefinition.DWmParameters.ELYTRON_ENABLED.getAttribute()).parseAndSetParameter(value, distributedWorkManagerOperation, reader);
+                                if (!distributedWorkManagerOperation.hasDefined(JcaDistributedWorkManagerDefinition.DWmParameters.ELYTRON_ENABLED.getAttribute().getName())) {
+                                    distributedWorkManagerOperation.get(JcaDistributedWorkManagerDefinition.DWmParameters.ELYTRON_ENABLED.getAttribute().getName()).set(new ModelNode(false));
+                                }
```

Comment:
Doing this test here doesn't account for the scenario where this element isn't present, which would be the typical way of configuring to use the default. Handling at the end of processing the entire [distributed-]workmanager element accounts for that; e.g. https://github.com/tadamski/wildfly/pull/4/commits/698ff398e98cbe68c31a0ce414df942439e1ecff#diff-e210f5446d28f4290a2dbc810897187203061441e0af68a8d403ad87afa7f6baR557

Reply:
@tadamski Assuming CI's ok for this branch, I'm going to go ahead and merge this. We can use a separate blocker JIRA to track domain transformation and legacy parsing aspects, which don't relate to EE 10 compliance.